lib/core/src/display/shape/text/text_field.rs

```diff
@@ -122,6 +125,7 @@ shared! { TextField
 
         /// Removes all cursors except one which is set and given point.
         pub fn set_cursor(&mut self, point:Vector2<f32>) {
+            self.word_occurrences = None;
```

Comment:
Minor one - I'm not sure if I would not create method `clear_word_occurences`. But I'm not sure why I have strong feelign it should be a method.